### PR TITLE
Update to match v1 API changes, add README.md

### PR DESCRIPTION
**Pull request by iridescent-beacon**
Probably easiest to go commit-by-commit here -- each is self explanatory but didn't want to make a ton of tiny PRs.

I do not plan on squash-merging this.

### CODE REVIEW

File: popup/p.html [hunk 1]
```diff
@@ -7,8 +7,11 @@
 </head>
 <body>
 <form id="loginform">
-  <label>Your UID: <input type="text" name="uid"></label>
-  <button type="button" name="btn">OK</button>
+  <label>Email: <input type="text" name="email"></label><br/>
+  <label>Password: <input type="text" name="password"></label><br/>
+  <label>Beta Key: <input type="text" name="beta_key"></label><br/>
+  <button type="button" name="btn">Login</button>
```
Comment (ypoluektovich) [inline popup/p.html]:
```suggestion
  <button type="button" name="btn">Log in</button>
```

File: popup/p.css [hunk 2]
```diff
@@ -27,3 +27,7 @@ body:not(.loggedin) #new {
 .tag[data-signal="false"] .sigAgainstContainer {
     font-weight: bold;
 }
+
+.tag[data-signal="false"] .sigEraseContainer {
```
Comment (ypoluektovich) [inline popup/p.css]: Shouldn't `data-signal` be something else here? Something like `null` or `undefined`.

File: popup/p.js [hunk 3]
```diff
@@ -109,27 +115,56 @@ function onLoggedOut() {
...
+      password: loginform.password.value,
+      betaKey: loginform.beta_key.value,
+    })
+  });
+  if (!res.ok) {
+    console.error(res);
+    // todo: show this in UI
+    throw 'failed to register';
+  }
+}
```
Comment (ypoluektovich) [inline popup/p.js:137]: No indication of a successful registration

File: popup/p.css (hunk 2 above)
Comment (iridescent-beacon) [reply popup/p.css]: Definitely, thanks for catching that :)

**Review by ypoluektovich — APPROVED** [review]

**Pull request merged**